src/linear_algebra/bilinear_form.lean

```diff
@@ -1453,4 +1453,71 @@ begin
   erw [add_right, show B m.1 y = 0, by rw hB₂; exact m.2 y hy, hm, add_zero]
 end
 
+section linear_adjoints
+
+lemma comp_left_apply_eq_unique (B₁ B₂ : bilin_form K V) (hB₂ : B₂.nondegenerate) {φ ψ : V →ₗ[K] V}
+  (hφ : ∀ v w, B₁ w v = B₂ (φ w) v) (hψ : ∀ v w, B₁ w v = B₂ (ψ w) v) : φ = ψ :=
```

Comment:
It surprised me that the order of `v` and `w` were swapped between the `∀` and `B`, or is this a convention?
```suggestion
  (hφ : ∀ v w, B₁ v w = B₂ (φ v) w) (hψ : ∀ v w, B₁ v w = B₂ (ψ v) w) : φ = ψ :=
```

Reply:
If we would like the right adjoint we need another definition of `dual_map` which requires another definition of `bilin_form.to_lin`. Actually, I'm not sure whether the statement is true since there is a chirality to `nondegenerate` in which we chose the definition to be a "left" one.